.gitignore

```diff
@@ -11,4 +11,4 @@
 api_tests/node_modules/*
 cmd/cmd
 
-public/
\ No newline at end of file
+public/
```

Comment:
I have concerns regarding ignoring the folder completely, are the dev builds going to work in this case? https://github.com/percona/everest/blob/c1224dc63cc0bab61e8ad7f47597726a4ebadc05/.github/workflows/dev-build.yaml#L37

Reply:
I'll resolve this one for now since we discussed that we will tackle this issue separately. For now I'm reverting the index.html back to the original (empty) state